showcase/app/templates/components/app-side-nav/index.hbs

```diff
@@ -45,9 +45,9 @@
   </Shw::Flex>
 
   <Hds::AppSideNav::Portal @targetName="sidenav-portal-demo-1" @ariaLabel="Primary on portal demo 1" as |Nav|>
-    <Nav.extraBefore><Shw::Placeholder @height="72px" @text="extraBefore" @background="#f3d9c5" /></Nav.extraBefore>
+    <Nav.ExtraBefore><Shw::Placeholder @height="72px" @text="extraBefore" @background="#f3d9c5" /></Nav.ExtraBefore>
```

Comment:
Note: this causes a small change in Percy because now the blocks are rendered (they weren't before)